diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -703,9 +703,10 @@ controller_interface::CallbackReturn DiffDriveController::configure_side(
   return controller_interface::CallbackReturn::SUCCESS;
 }
 
-bool DiffDriveController::on_set_chained_mode(bool /*chained_mode*/)
+bool DiffDriveController::on_set_chained_mode(bool chained_mode)
 {
   // Always accept switch to/from chained mode
+  (void)chained_mode;
```

Comment:
I know I proposed this, but I'm not sure if we should add this here :see_no_evil:

Reply:
maybe this is why the weird
`return true || chained_mode;`
is there

Reply:
Semantically it doesn't make sense as if you use true || something. It will never even use it. 
Anyway, if that's the only way. Let's use it

Reply:
I had a chat with my AI colleague.
It offered me the solution with (void), but also this one
```
bool DiffDriveController::on_set_chained_mode([[maybe_unused]] bool)
{
  // Always accept switch to/from chained mode
  return true;
}
```
all three work, but let's use the one with || and maybe change all chainable controllers at once if we want to change that.